src/kube-runtime/src/init

```diff
@@ -56,6 +86,11 @@ mkdir -p ${PAI_LOG_DIR}
 ${PAI_INIT_DIR}/frameworkbarrier > ${PAI_LOG_DIR}/barrier.log 2>&1
 echo "barrier returns $?" >> ${PAI_INIT_LOG_FILE}
```

Comment:
Tee all runtime logs to docker/k8s stdout/err for easy debugging?

Reply:
If we want to redirect lgos to stdout & stderr, I think it's better to wrap the init container or change the init container start command.

I prefer change the docker start command

---
In reply to: [323551280](https://github.com/microsoft/pai/pull/3585#discussion_r323551280) [](ancestors = 323551280)

Reply:
Fixed by this PR: [https://github.com/microsoft/pai/pull/3620](https://github.com/microsoft/pai/pull/3620)

---
In reply to: [323602539](https://github.com/microsoft/pai/pull/3585#discussion_r323602539) [](ancestors = 323602539,323551280)